src/network.ts

```diff
@@ -0,0 +1,47 @@
+import { CreateNetworkOptions } from "./docker-client";
+import { DockerodeClientFactory } from "./docker-client-factory";
+import { RandomUuid } from "./uuid";
+
+class StartedNetwork {
+  constructor(
+    private readonly id: string,
+    private readonly options: CreateNetworkOptions,
+    private readonly dockerClientFactory: DockerodeClientFactory
+  ) {}
+
+  public getId(): string {
+    return this.id!;
```

Comment:
Is the `!` required?

Reply:
I wasn't sure how to resolve TypeScript complaint here - the problem is that in the beginning `id` is undefined, so it requests to support that case in typings as well I guess.